ui/src/conferences/components/ProceedingsAction.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List, Map } from 'immutable';
+import { Menu } from 'antd';
+
+import ExternalLink from '../../common/components/ExternalLink';
+import ActionsDropdownOrAction from '../../common/components/ActionsDropdownOrAction';
+import IconText from '../../common/components/IconText';
+import JournalInfo from '../../common/components/JournalInfo';
+
+function renderProceedingsDropdownAction(proceeding, index) {
+  const href = proceeding.getIn(['record', '$ref']);
+  const publicationInfo = proceeding.getIn(['publication_info', 0], Map());
+  return (
+    <Menu.Item key={href}>
```

Comment:
what about the arXiv part? I thought that if there is an arXiv then we need to display that, otherwise display the publication info

Reply:
After discussion, we decided to remove that part. there's an updated mockup in the issue